src/Symfony/Component/Notifier/Bridge/SmsFactor/composer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "symfony/smsfactor-notifier",
```

Comment:
```suggestion
    "name": "symfony/smsfactor-notifier",
```
@fabpot are you fine with the package name?

Reply:
Must be `sms-factor-notifier` as the class name is `SmsFactor` and not `Smsfactor`